engine/database_connection_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/database/drivers"
 )
 
-func CreateDatabase(t *testing.T) {
+func CreateDatabase(t *testing.T) { //nolint
```

Comment:
Please be more specific with your nolint usage. Here is an example from our codebase:
`//nolint:forcetypeassert // Not necessary from a pool`
we specify what we are nolinting and we provide a comment on why we are nolinting